src/circuit/gadget/sinsemilla.rs

```diff
@@ -337,12 +337,32 @@ mod tests {
                 meta.advice_column(),
             ];
 
-            let constants = meta.fixed_column();
+            // TODO: Replace with public inputs API
+            let constants_1 = [
+                meta.fixed_column(),
+                meta.fixed_column(),
+                meta.fixed_column(),
+                meta.fixed_column(),
+                meta.fixed_column(),
+                meta.fixed_column(),
+            ];
+            let constants_2 = [
+                meta.fixed_column(),
+                meta.fixed_column(),
+                meta.fixed_column(),
+                meta.fixed_column(),
+                meta.fixed_column(),
+                meta.fixed_column(),
+            ];
+            let constants_3 = meta.fixed_column();
```

Comment:
Where is this used?

Reply:
This isn't used in the current PR and has been removed. However, after we merge the changes to `LookupRangeCheckConfig`, this will be used to configure the ECC chip.